source/switching_to_live/set_up_a_live_epdq_account/index.html.md.erb

```diff
@@ -198,11 +200,15 @@ In your [GOV.UK Pay
 
 5. Select the test transaction and check if you can refund it.
 
-> The refund option may take up to 20 minutes to appear after submitting the transaction.
+    The refund option may take up to 20 minutes to appear after submitting the transaction.
+
+    If you have not [set up notifications](#set-up-notification-settings) correctly, some features such as refunds will not be available.
```

Comment:
I’ve thought about this a bit more and realised that notifications not being set up correctly won’t stop the refund button showing up for ePDQ (it can for other gateways though).

Reply:
will remove for epdq